app/src/main/java/com/codingblocks/cbonlineapp/activities/HomeActivity.kt

```diff
@@ -369,7 +405,69 @@ class HomeActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelecte
     @TargetApi(25)
     private fun removeShortcuts() {
         val shortcutManager = getSystemService(ShortcutManager::class.java)
-        shortcutManager.disableShortcuts(Arrays.asList("shortcut1"))
+       // shortcutManager.disableShortcuts(Arrays.asList("shortcut1"))
+        shortcutManager.disableShortcuts(Arrays.asList("topcourse"))
         shortcutManager.removeAllDynamicShortcuts()
     }
+
+    fun isNetworkAvailable(context: Context): Boolean {
```

Comment:
This function was shifted to NetworkUtils. You can use this like NetworkUtils.isNetworkAvailable(this)